roles/confluent.variables_handlers/vars/main.yml

```diff
@@ -10,7 +11,7 @@ zookeeper_default_user: cp-kafka
 zookeeper_default_group: confluent
 zookeeper:
   config_file: /etc/kafka/zookeeper.properties
-  systemd_file: /usr/lib/systemd/system/{{zookeeper_service_name}}.service
+  systemd_file: "{{default_systemd_dir}}/{{zookeeper_service_name}}.service"
```

Comment:
if we are only doing this feature for connect do we need this bit overwriteable for zk or kafka for example?